docs/02-connect/04-eon_block_explorer.md

```diff
@@ -0,0 +1,584 @@
+
+The Gobi Testnet Explorer highlights the Fee Payment transactions:
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-fee.png").default} alt="fee-payment" width="900" height="700" />
+</p>
+
+Key information includes:
+
+- **Address** - The wallet address receiving the fee payment. 
+- **Amount of tZEN** - The amount of tZEN transferred in the transaction.
+- **Block Number** - The number assigned to the block.
+- **Elapsed timer for the block** - A timer that displays the elapsed time since block creation period.
+
+# Blockchain Menu
+
+The Blockchain menu has two feature sections:
+
+- Blocks
+- Transactions
+
+## Blocks 
+The Blocks section includes:
+
+- Blocks
+- Forged Blocks (Reorg)
+
+### Forged Blocks (Reorgs)
+Forged blocks or chain reorganization (reorgs) takes place when a block is removed from the blockchain and a node receives blocks that are part of a new (longest) chain.
+
+The Gobi Testnet Explorer highlights the Reorgs:
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-reorg.png").default} alt="reorgs" width="900" height="700" />
+</p>
+
+Key information includes:
+
+- **Block Number** - The number assigned to the block.
+- **Transactions** - The number of transactions for the block.
+- **Bytes** - The number of bytes for the block.
+- **Elapsed timer for the block** - Clock counting since block creation period.
+- **Miner (wallet) address** - The miner's wallet address. Click the address to **Address Detail** page.
+- **Priority  Fee** - The priority fee is the additional amount of fee to have your transaction complete faster. This is also known as a tip.
+- **Reward Pool Fee** - The total amount of required fee users paid for the transactions in this block. This is also known as base fee. 
+- **Gas Limit** - The maximum amount of total gas consumed for all transactions in one block.
+- **Gas Used** - The total amount of gas used by all transactions in this block.
+
+## Transactions  
+The Transactions section includes:
+
+- Validated 
+- [Forward Transfer](#forward-transfer) 
+- [Fee Payments](#fee-payment)
+- Pending
+- Verified Contracts
+
+### Validated Transactions
+Transaction validation indicates that a transaction conforms to a specific protocol requirement to deem it as valid. Once the transaction is considered valid it is added to the distributed ledger.
+ 
+The **Validated Transaction** feature provides key information about the transaction.
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-validated.png").default} alt="validated-transaction" width="900" height="700" />
+</p>
+
+
+Key information includes:
+
+- **Transaction Status** - Success or Error: (Awaiting internal transaction for reason)
+- **Transaction ID (hash)** - The unique identifier that validates the transaction.
+- **Address** - The wallet address initiating the transaction. 
+- **Transaction Fee** - The amount charged to process the transaction.
+- **Contract Address** - Click the address to Address Detail page.

```

Comment:
edit wording

Reply:
Also for consistency, remember to describe what the contract address is for

Reply:
made corrections